lab1/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="app.css">
+	<title>Youtube Famous</title>
+</head>
+<body>
+		<header>
+			<h1>View Collector</h1>
+		</header>
+		
+		<div class="story">
+			<h4>The Story Goes Here</h4>
```

Comment:
should contain at least 3 stories.

Reply:
h4 is not recommended here as the ADA suggests the header has to follow the order of 1 `h1` in the whole page and the second header should be `h2`